README.md

```diff
@@ -13,11 +14,11 @@ workspaces with a single click.
 - Works in air-gapped or restricted networks. Just connect to your Coder
   deployment!
 - Supports multiple editors: VS Code, Cursor, and Windsurf.
-  > [!NOTE]  
-  > The extension builds on VSCode provided implementations of SSH. Make sure
-  > you have the correct ssh extension installed for your editor
-  > (ms-vscode-remote.remote-ssh or codeium.windsurf-remote-openssh for
-  > windsurf)
+
+> [!NOTE]
+> The extension builds on VSCode provided implementations of SSH. Make
```

Comment:
```suggestion
> The extension builds on VS Code-provided implementations of SSH. Make
```

Reply:
I see a mix of usage of VSCode and VS Code. Can we set a linter rule to enforce the correct `VS Code`?